docs/products/kafka/howto/request-access-topic.md

```diff
@@ -26,10 +29,31 @@ You can view the service user and ACLs in the following locations in the
 
 ## Prerequisites
 
-[Governance](/docs/products/kafka/howto/enable-governance) enabled for your organization.
+<Tabs groupId="group1">
+<TabItem value="console" label="Console" default>
+
+[Governance](/docs/products/kafka/howto/enable-governance) enabled for your organization
```

Comment:
Would this also be a requirement for TF?

Reply:
Good catch, added this.

Reply:
From my understanding, Governance does not need to be enabled when using Terraform. The approval checker operates independently and does not interact with the Aiven Console or aiven-core. It also does not rely on any Governance settings.